htdocs/core/lib/functions.lib.php

```diff
@@ -12154,7 +12166,15 @@ function getElementProperties($element_type)
 		'dir_output' => $dir_output
 	);
 
-	//var_dump($element_properties);
+	$reshook = $hookmanager->executeHooks('getElementProperties', $parameters);
+	if ($reshook < 0) {
+		setEventMessages($hookmanager->error, $hookmanager->errors, 'errors');
```

Comment:
The method is getElementProperties is low level method that can be run out of user/web context (in cron job for exemple).
So we should not have setEventMessage inside.